app/presenters/trashed_revision_presenter.rb

```diff
@@ -0,0 +1,73 @@
+class TrashedRevisionPresenter < BasePresenter
+  presents :revision
+
+  def created_at_ago
+    h.local_time_ago(revision.created_at)
+  end
+
+  def info
+    [
+      icon,
+      title,
+      type.downcase,
+      location,
+    ].join(" ").html_safe
+  end
+
+  private
+
+  def icon
+    # Set icon css depending on object type.
+    icon_css = %w{fa}
+    icon_css << case type
+                when 'Evidence'
+                  'fa-flag'
+                when 'Issue'
+                  'fa-bug'
+                when 'Note'
+                  'fa-file-text-o'
+                else
+                  ''
+                end
+    h.content_tag(:i, '', class: icon_css)
+  end
+
+  def location
+    # Get node if object is a Note or an Evidence.
+    if ['Note','Evidence'].include?(type)
+      if trashed_object.node
+        "at " + h.link_to(trashed_object.node.label, trashed_object.node)
+      else
+        'at a Node which has since been deleted'
+      end
+    else
+      ''
+    end
+  end
+
+  def title
+    # Get title or content first characters.
+    if trashed_object.fields.empty?
```

Comment:
> we're assuming that the trashed object has fields

A safe assumption for now, since the only items that will appear in the trash are `Issue`, `Note`, and `Evidence`, which all have fields.

But the problem with using `HasFields#title` here is that if there isn't a title we'll get the ugly string "This (class) doesn't provide a #[Title]# field", which ends up looking like this:

<img width="486" alt="screen shot 2016-09-07 at 16 36 20" src="https://cloud.githubusercontent.com/assets/2656485/18315995/45efe280-7519-11e6-8442-f517108d0e4d.png">

I think the current approach looks better. What do you think?